talent/application-process.md

```diff
@@ -64,4 +64,6 @@ The interviews consist of the following:
 
 - At source{d} we believe that amazing talent should not be constrained by physical barriers. Our multinational and multicultural team shares a culture of openness and excellence and passion for programming.
 
-- At the moment, we are 25+ people from 10 different countries working closely together from our office in Madrid. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely. Currently, we have remote team members in Portugal, Estonia & Russia.
+- At the moment, we are 25+ people from 10 different countries working closely together from our office in Madrid. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely. Currently, we have remote team members in Portugal, Estonia & Russia. 
```

Comment:
since @creachadair @campoy @kuba-- @meyskens and @m09 joined us...
and I don't knot where @mcarmonaa is working from, but he is not doing it from Spain as far as I know..
```suggestion
- At the moment, we are 25+ people from 10 different countries working closely together from our office in Madrid (Spain). We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely. Currently, we have remote team members in USA, Portugal, France, Belgium, Poland, Estonia and Russia. 
```

Reply:
@dpordomingo thanks for the update. As a suggestion: I would remove the ```(Spain)``` as we have already mentioned our offices in our careers page and I would add SF as well (I changed the job description a few days ago as location: ```Madrid, SF, remote```

- At the moment, we are 25+ people from 10 different countries working closely together from our offices in Madrid and San Franscisco. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our offices, but you may also choose to work remotely. Currently, we have remote team members in USA, Portugal, Ireland, France, Belgium, Poland, Estonia and Russia.

Reply:
@creachadair Thanks for the suggestion. In a previous call with @eiso he also suggested to include in our job descriptions at the ```location section``` Seattle as well, but on our [website-offices](https://sourced.tech/company/#offices) we mention the offices of SF and Madrid only, so I thought it won't be consistent and also candidates could ask about an office that is not a proper office yet. 

That makes me a bit hesitant, what do you think @eiso ?

Reply:
That's a good point.